source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -245,6 +247,45 @@ content: |
            To use Google Cloud KMS, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
            version 1.1 or later in your application's environment.
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           // _connectionString is defined elsewhere as "mongodb://localhost:27017"
+
+           var keyVaultClient = new MongoClient(_connectionString);
+           var clientEncryptionOptions = new ClientEncryptionOptions(
+               keyVaultClient: keyVaultClient,
+               keyVaultNamespace: _keyVaultNamespace,
+               kmsProviders: kmsProviders);
+
+           var clientEncryption = new ClientEncryption(clientEncryptionOptions);
+
+
+           var gcpDataKeyProjectId = Environment.GetEnvironmentVariable("FLE_GCP_PROJ_ID");
+           var gcpDataKeyLocation = Environment.GetEnvironmentVariable("FLE_GCP_KEY_LOC"); // Optional. e.g. "global"
+           var gcpDataKeyKeyRing = Environment.GetEnvironmentVariable("FLE_GCP_KEY_RING");
+           var gcpDataKeyKeyName = Environment.GetEnvironmentVariable("FLE_GCP_KEY_NAME");
+           var gcpDataKeyKeyVersion = Environment.GetEnvironmentVariable("FLE_GCP_KEY_VERSION"); // Optional
+           var gcpDataKeyEndpoint = Environment.GetEnvironmentVariable("FLE_GCP_KMS_ENDPOINT"); // Optional, KMS URL, defaults to https://www.googleapis.com/auth/cloudkms
```

Comment:
Make sure that all env variables match between different documents

Reply:
Thanks. These are all the same everywhere now.